braindecode/datasets/base.py

```diff
@@ -142,6 +144,88 @@ def transform(self, value):
         self._transform = value
 
 
+class SequenceDataset(WindowsDataset):
+    """Dataset of sequences of consecutive windows.
+
+    Parameters
+    ----------
+    windows : mne.Epochs
```

Comment:
would it make sense to rather construct it from a WindowsDataset ?

Reply:
Good point, yes I think this might be better. If we decide to stick with this approach (see https://github.com/braindecode/braindecode/pull/263#issuecomment-866009957) then I can do the change.